wemake_python_styleguide/visitors/ast/builtins.py

```diff
@@ -478,7 +510,7 @@ def _check_set_elements(
             # Non-constant nodes raise ValueError,
             # unhashables raise TypeError:
             with suppress(ValueError, TypeError):
-                # Similar value:
+                # Similar val:
```

Comment:
Was there a problem with this comment? I am asking in fear of any false positives.